src/components/scenes/LoginScene.tsx

```diff
@@ -75,11 +71,6 @@ export function LoginSceneComponent(props: Props) {
   // Effects
   // ---------------------------------------------------------------------
 
-  const [backgroundImage = theme.backgroundImage] = useAsyncValue<ImageProp | undefined>(async () => {
-    const url = pickRandom(theme.backgroundImageServerUrls)
-    return await getBackgroundImage(disklet, url, theme.backgroundImage)
-  }, [disklet, theme])
```

Comment:
What if we could pass a "background component" to the app as part of the infra for white-labeling etc. The `DotsBackground` component just becomes a parameter to the app defined perhaps in the appConfig or something.

Can a component be a value for a context provider which all the dependents could access to render the appropriate component. This is a bit uncommon of a pattern, but I did consider trying out this pattern for the dynamic search bar to sort of "portal" the scene's drawer component (render function) to the MenuTabs component which is rendered in a disparate part of the component tree.

Reply:
This is a good idea, but maybe out of scope for this PR.